nextflow.config

```
@@ -114,6 +114,10 @@ params {
     skip_deduplication = false
     deduplication_tool = 'markduplicates'
 
+    // MtNucRatio
+    run_mtnucratio = false
+    mtnucratio_header = 'MT'
```

CAn you rename this 
```suggestion
    mitochondrion = 'MT'
```

AS thisi s what Iv'e called it in #952 - or we could rename it to `mitochondrion_header`... thoughts @TCLamnidis ?

---

Lets say header to make sure it's linked with the FASTA format, I can update my PR accordingly